src/smart-components/Thread/components/ThreadList/index.scss

```diff
@@ -0,0 +1,20 @@
+.sendbird-thread-list-item .sendbird-separator {
+  margin: 4px 0px;
+}
+
+.sendbird-thread-list .sendbird-message-content__middle__sender-name {
+  white-space: nowrap;
+  max-width: 210px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.sendbird-thread-list .sendbird-message-content.incoming .sendbird-message-content__middle {
+  max-width: 200px;
+}
+
+.sendbird-thread-list .sendbird-message-content__middle__message-item-body.sendbird-thumbnail-message-item-body.outgoing {
```

Comment:
simplify these CSS rules ~~
Probably we just need `sendbird-thumbnail-message-item-body.outgoing` 
Maybe I am wrong(?)

Reply:
It works! thanks